SwiftRuntimeLibrary.Mac/SwiftRuntimeLibrary.Mac.csproj

```diff
@@ -284,8 +284,7 @@
     <Reference Include="System" />
   </ItemGroup>
   <Target Name="GeneratedCSCode" BeforeTargets="CoreCompile" Inputs="$(MSBuildProjectFullPath)" Outputs="GeneratedCode\BindingMetadata.MacOS.cs">
-    <Exec Command="mkdir -p GeneratedCode" />
-    <Exec Command="/Library/Frameworks/Mono.framework/Versions/Current/Commands/mono $(MSBuildProjectDirectory)\..\type-o-matic\bin\Debug\type-o-matic.exe --swift-lib-path=$(MSBuildProjectDirectory)/../apple/build/Ninja-ReleaseAssert/swift-macosx-x86_64/lib/swift/ --platform=mac --xamglue-framework=$(MSBuildProjectDirectory)/../swiftglue/bin/$(Configuration)/mac/FinalProduct/XamGlue.framework --output=GeneratedCode\BindingMetadata.MacOS.cs" />
+    <Exec Command="cd ../SwiftRuntimeLibrary ; make create-pinvokes-mac configuration=$(Configuration)" />

```

Comment:
There is not need to do a `cd` here. Take a look at the -C option in make.

> -C dir, --directory=dir
            Change to directory dir before reading the makefiles or doing anything else.  If multiple -C options are specified, each is  interpreted  relative  to
            the previous one: -C / -C etc is equivalent to -C /etc.  This is typically used with recursive invocations of make.